appserver/tests/appserv-tests/devtests/web/jspBomInClassicSyntax/docroot/UTF-8.jsp

```diff
@@ -1 +1,43 @@
-﻿this is a test
```

Comment:
All of these UTF-* files should probably not be modified.  I assume they have very specific UTF-* formatted data.  They're small enough that they don't need a copyright header.

Reply:
Reverted.